libs/blocks/global-footer/global-footer.js

```diff
@@ -69,17 +69,25 @@ class Footer {
 
     // TODO: log to LANA if Footer content could not be found
     if (!this.body) return;
-    // TODO: revisit region picker and social links decoration logic
+
     const regionAnchor = this.body.querySelector('.region-selector a');
-    if (regionAnchor?.href) {
-      regionAnchor.setAttribute('href', `${regionAnchor.getAttribute('href')}#_dnt#_dnb`);
-    }
-    const socialLinks = document.querySelectorAll('.social a');
-    socialLinks.forEach((socialLink) => {
-      socialLink.setAttribute('href', `${socialLink.getAttribute('href')}#_dnb`);
-    });
+    const regionParent = regionAnchor?.parentNode;
+    const detachedRegion = regionParent?.removeChild(regionAnchor);
+
+    const socialLinks = [...this.body.querySelectorAll('.social a')];
+    const socialParents = socialLinks.map((link) => link.parentNode);
+    const detachedSocialLinks = socialLinks.map(
+      (link, index) => socialParents[index].removeChild(link),
+    );
+
     decorateLinks(this.body);
 
+    if (detachedRegion && regionParent) regionParent.appendChild(detachedRegion);
+
+    detachedSocialLinks.forEach((link, index) => {
+      if (link && socialParents[index]) socialParents[index].appendChild(link);
+    });
+
```

Comment:
Detaching the links momentarily could be a solution to this, but I feel this is a bit too verbose. Could we not simplify with something like:
```
const [ region, social ] = ['.region-selector', '.social'].map((selector) => this.body.querySelector(selector));
const [ regionParent, socialParent ] = [ region.parentElement, social.parentElement ];
[ regionParent, socialParent ].forEach((parent) => parent.replaceChildren());

decorateLinks(this.body);

regionParent.appendChild(region);
socialParent.appendChild(social);
```

Reply:
Done. Indeed is simpler like this